ibm_cloud_sdk_core/base_service.py

```diff
@@ -35,13 +35,19 @@
                     cleanup_values, read_external_sources, strip_extra_slashes)
 from .version import __version__
 
-logger = logging.getLogger(__name__)
-
 # Uncomment this to enable http debugging
 # import http.client as http_client
 # http_client.HTTPConnection.debuglevel = 1
 
 
+logger = logging.getLogger(__name__)
+
+
+#pylint: disable=too-many-instance-attributes
+#pylint: disable=too-many-locals
+class BaseService:
+
+
 #pylint: disable=too-many-instance-attributes
 #pylint: disable=too-many-locals
 class BaseService:
```

Comment:
@datawookie I should have said that my suggestion was invalid in that context, because I wasn't able to select the whole code block.
You can easily fix the duplicated code manually, then check it as @padamstx mentioned.